src/redux/actions/auth/authActions.js

```diff
@@ -20,9 +20,16 @@ const loginAction = ({ username, password }) => (dispatch) => {
     return dispatch(login(response));
   }).catch((error) => {
     if (error.response) {
-      return dispatch(loginFailed(error.response.data.detail));
+      let errorMessage = error.response.data.detail;
+      if (errorMessage === 'Not found.') {
+        errorMessage = 'No such user was found. Please signup instead.';
```

Comment:
@collinewait  initially it was done like that but @Duncanian suggested different messages. This is because admin type users and normal users return different error messages from the server. I think Admin users always exist on the server (was pre-set up) and would not return "Not found". Normal users may or may not exist and return "Not found"... we therefore need to inform them that they should sign up.

Reply:
I think `No such user was found` is enough. Suggesting signup wont be necessary e.g when the username has a typo.

Reply:
@Tittoh Let me change it to that.